mscore/file.cpp

```diff
@@ -1602,6 +1602,7 @@ void MuseScore::exportFile()
 #endif
       fl.append(tr("Standard MIDI File") + " (*.mid)");
       fl.append(tr("MusicXML File") + " (*.xml)");
+      fl.append(tr("MusicXML 3.1+ File") + " (*.musicxml)");
```

Comment:
Well, dropping .xml as an export format extension might be OK for master, but IMHO won't for 2.2